calendar/calendar.yaml

```diff
@@ -509,4 +494,20 @@
       Passcode: 273734
 
       If you want to be manually added to the invite link, please reach out to Amber Graner (akgraner)
-  organizer: akgraner
\ No newline at end of file
+  organizer: akgraner
+
+- id: kf025
+  name: Kubeflow Pipelines Community Meeting (PST PM)
+  date: 08/05/2020
+  time: 5:30pm-6:10pm
+  frequency: every-4-weeks
+  video: https://meet.google.com/jkr-dupp-wwm
+  until: 07/03/2024
+  attendees:
+    - email: kubeflow-discuss@googlegroups.com
+    - email: kubeflow-pipelines@google.com
+  description:
+    - |
+      Notes & Agenda: http://bit.ly/kfp-meeting-notes
+      Join at: https://meet.google.com/jkr-dupp-wwm
+  organizer: chensun
```

Comment:
Can we remove this meeting to avoid duplication?

Reply:
This is part of instructions to remove recurring meetings. We cannot just remove it from the file, but move to the bottom of the file and set a `until` field with a date in the past. If we don't do that, the calendar entry will never be removed.

Reply:
I see, thank you for the reminder!